napari/_qt/qt_viewer.py

```diff
@@ -90,13 +90,14 @@ class QtViewer(QSplitter):
     """
 
     def __init__(self, viewer: Viewer, welcome=False):
```

Comment:
I know this was already here before you worked on this feature, but what do you think about renaming this variable name so that it is more clear that it represents a boolean? Maybe also something that could be more general to use in future features as well like "isEmpty", "isBlank"?

Reply:
How about:

```python
def __init__(self, viewer: Viewer, show_welcome_screen: bool = False): 
```
?

Reply:
I think that's better. Thanks

Reply:
Fixed!